jmxtrans-cluster/src/main/java/com/googlecode/jmxtrans/cluster/ClusterManager.java

```diff
@@ -0,0 +1,48 @@
+package com.googlecode.jmxtrans.cluster;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.inject.Guice;
+import com.google.inject.Inject;
+import com.google.inject.Injector;
+import org.apache.commons.configuration.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+
+/**
+ * ClusterManager Tester.
+ *
+ * @author Tibor Kulcsar
+ * @version 1.0
+ * @since <pre>May 17, 2016</pre>
+ */
+public class ClusterManager {
+    private static final Logger log = LoggerFactory.getLogger(ClusterManager.class);
```

Comment:
I'm not sure why you need this adapter class. It just renames the methods from `ClusterService` without bringing any additional logic. Couldn't the clients use `ClsuterService` directly?